src/layout/DefaultLayout/DefaultLayout.test.tsx

```diff
@@ -66,9 +66,7 @@ describe('DefaultLayout component', () => {
   })
 
   it('renders personal data', () => {
-    expect(screen.getByTestId('personal-data').textContent).toContain(
-      'Welcome, BERNADETT'
-    )
+    expect(screen.getByTestId('personal-data').textContent).toContain('Welcome')
   })
```

Comment:
I added a `useState` hook into `DefaultLayout` to manage the user's `displayName`. The default value for the state is an empty string, which causes `PersonalData` to just display 'Welcome!' We have several e2e tests that check the `displayName` value, so rather than going through testing `useState` here, I opted to just verify the default message.